pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -62,6 +65,7 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	cmd.Flags().VarP(&config.ImageStreams, "image", "i", "Name of an image stream to to use as a builder.")
 	cmd.Flags().Var(&config.DockerImages, "docker-image", "Name of a Docker image to use as a builder.")
 	cmd.Flags().StringVar(&config.Name, "name", "", "Set name to use for generated build artifacts")
+	cmd.Flags().VarP(&config.Environment, "env", "e", "Specify key value pairs of environment variables to set into each builder container.")
```

Comment:
You're right... hmm, I thought these would go into the output image. I would argue that this is not very useful, since we only whitelist the LOG_LEVEL var to pass to the builder: 
https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/util.go#L22